bbbackend/models/user.js

```diff
@@ -19,11 +26,19 @@ const UserSchema = new mongoose.Schema(
       type: String,
       required: false,
     },
+    groups: {
+      type: Object,
+      required: false,
+    },
     friends: {
       type: Object,
       required: false,
     },
-    lists: {
```

Comment:
See above comment. Make sure the itemList reflects our schema to include item objects.